src/placeos-frontend-loader/loader.cr

```diff
@@ -43,7 +43,7 @@ module PlaceOS::FrontendLoader
       @content_directory : String = Loader.settings.content_directory,
       @update_crontab : String = Loader.settings.update_crontab
     )
-      @actioner = PlaceOS::FrontendLoader::GitHubRemote.new
+      @actioner = PlaceOS::FrontendLoader::Github.new
```

Comment:
I think the final step here is matching the remote ref to the correct loader in `load`

Reply:
A semi-clean way would be creating an instance method `Loader#remote(ref : Reference) : Remote` that gets the correct actioner for the ref then you pass that to the `Loader.load` method

You could just raise if it's not supported for now